sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -155,8 +155,6 @@ public static DataSourceConfiguration create(
         String driverClassName, String url, String username, String password) {
       checkNotNull(driverClassName, "driverClassName");
       checkNotNull(url, "url");
```

Comment:
* Is it the case that even the username can be null, when the configuration is specified by driver class name / url / username / password?
* Please mark the nullable parameters of the function as @Nullable.